packages/e2e-tests/tests/executors.spec.ts

```diff
@@ -39,9 +40,55 @@ test(`Create custom container executor`, async ({page}) => {
   await api.removeExecutor(realExecutorName);
 });
 
-test.skip(`Custom container executor - general settings`, async ({page}) => {});
+test(`Custom container executor - general settings`, async ({page}) => {
+  const executorName = 'container-executor-curl-1';
+  const executorData = testDataHandler.getExecutor(executorName);
+  const realExecutorName = executorData.name;
+  await api.assureExecutorCreated(executorData);
+
+  const mainPage = new MainPage(page);
+  await mainPage.visitMainPage();
+
+  const navigationSiderPage = new NavigationSiderPage(page);
+  await navigationSiderPage.openMenuItem('executors');
+
+  const executorsPage = new ExecutorsPage(page);
+  await executorsPage.openExecutorSettings(realExecutorName);
+
+  const executorGeneralSettingsPage = new ExecutorGeneralSettingsPage(page);
+  const executorNameInput = await executorGeneralSettingsPage.getExecutorName();
```

Comment:
Usually it's better to user "validators" instead of getting and then comparing the value.
The difference is in this case it's waiting for element matching the selector (`input[id="general-settings-name-type_name"]`), then getting it's value, and then comparing (expect assertion). Element will be checked once, so any state transitions can make the test unstable.

In case of validators it would be looking for the same element, but with specific text value.
So, `executorGeneralSettingsPage.validateExecutorName(executorName)`:
and, all that's needed is checking if the element exists:
```
this.page.locator(`xpath=//input[@id="general-settings-name-type_name" and text="${executorName}"]`)
```


Additionally, on the test level I would use higher-level function instead - like `executorGeneralSettingsPage.validateExecutorDetails` that would then contain `validateExecutorName` on the Page level. The main goal is to keep tests high-level, and keep all of the details on the lower (Page) level.